src/masternodes/rpc_accounts.cpp

```diff
@@ -1802,6 +1810,30 @@ UniValue getburninfo(const JSONRPCRequest& request) {
     return result;
 }
 
+UniValue listcustomtxtypes(const JSONRPCRequest& request) {
+    RPCHelpMan{"listcustomtxtypes",
+               "\nList all available custom transaction types.\n",
+               {
+               },
+               RPCResult{
+                       "{\"1\": \"ICXCreateOrder\", \"2\": \"ICXMakeOffer\", ...}     (object) List of custom transaction types { [single letter representation]: custom transaction type name}\n"
+               },
+               RPCExamples{
+                       HelpExampleCli("listcustomtxtypes", "")
+                       + HelpExampleRpc("listcustomtxtypes", "")
+               },
+    }.Check(request);
+
+    UniValue typeObj(UniValue::VOBJ);
```

Comment:
list* -- Are we not converging to use arrays for list in this RPC version?

Reply:
But the output makes sense to be in an array. Wonder if we should just call it `getdefitxtypes`?

Reply:
I wondered as well. It made sense to be called list and I initially did it with array but it's much more natural as an object. 
`getdefitxtypes` sounds good and is consistent with list* being only arrays

Reply:
`getdftxtypes` ?

Reply:
If I recall, the conclusion here was to use `getcustomtxcodes` ?